src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -171,6 +171,51 @@
   <data name="AbstractTypesShouldNotHaveConstructorsMessage" xml:space="preserve">
     <value>Abstract type '{0}' should not have public constructors</value>
   </data>
+  <data name="LoggerMessageDiagnosticNumericsInFormatStringTitle" xml:space="preserve">
+    <value>Numerics should not be used in logging format string</value>
+  </data>
+  <data name="LoggerMessageDiagnosticNumericsInFormatStringMessage" xml:space="preserve">
+    <value>Numerics should not be used in logging format string</value>
+  </data>
+  <data name="LoggerMessageDiagnosticNumericsInFormatStringDescription" xml:space="preserve">
+    <value>Numerics should not be used in logging format string.</value>
```

Comment:
Please rereview and update messaging as mentioned in the API review, for example these could be something like this:
```suggestion
  <data name="LoggerMessageDiagnosticNumericsInFormatStringTitle" xml:space="preserve">
    <value>Named placeholders are preferred than numeric in logging format string</value>
  </data>
  <data name="LoggerMessageDiagnosticNumericsInFormatStringMessage" xml:space="preserve">
    <value>Use named placeholders instead numeric placeholders</value>
  </data>
  <data name="LoggerMessageDiagnosticNumericsInFormatStringDescription" xml:space="preserve">
    <value>Using named placeholders in logging format string is more descriptive than numeric placeholders.</value>
```